packages/communication-react/src/index.ts

```diff
@@ -229,7 +229,8 @@ export type {
 } from '../../react-components/src';
 /* @conditional-compile-remove(PSTN-calls) */
 export type { HoldButtonProps, HoldButtonStrings, VideoTileStrings } from '../../react-components/src';
-
+/* @conditional-compile-remove(call-readiness) */
```

Comment:
Given the calling SDK may not support this for V1 call readiness, I would perhaps make a new conditional-compile for this feature called unsupported-browser

Reply:
I'm guessing this is for the other PR?

Reply:
lol yep 😅